doc/examples/bundle_profiles.py

```diff
@@ -0,0 +1,183 @@
+"""
+========================================================
+Tractometry: extracting profiles from segmented bundles
+========================================================
+
+In this example, we will we will extract the values of a statistic from a
+volume, using the coordinates along the length of a bundle. This is called
+tractometry.
```

Comment:
This is a type of tractometry. 
Tractometry does not always measure along the the length of bundles.

Reply:
What else does tractometry do?

I think the term comes from this ISMRM presentation: https://cds.ismrm.org/protected/11MProceedings/files/678.pdf 

That's also the way it's used in this paper: https://www.frontiersin.org/articles/10.3389/fninf.2017.00042/full

Reply:
I've changed this, but I still think that tractometry is the right term here. Let me know what you think.